funnel/models/project.py

```diff
@@ -927,6 +927,30 @@ def is_saved_by(self, user):
 )
 
 
+@Project.features('has_rsvp')
+def project_has_rsvp(obj):
+    return (
+        not obj.schedule_state.PAST
+        and obj.schedule_state.PUBLISHED
+        and (
+            obj.boxoffice_data is None
+            or 'item_collection_id' not in obj.boxoffice_data
+            or not obj.boxoffice_data['item_collection_id']
+        )
```

Comment:
All this is cheaper than `PAST` which requires a column property and may involve a db roundtrip. Maybe put the `PAST` condition last.